sql/core/src/test/scala/org/apache/spark/sql/execution/vectorized/ColumnarBatchSuite.scala

```diff
@@ -1261,4 +1261,140 @@ class ColumnarBatchSuite extends SparkFunSuite {
     batch.close()
     allocator.close()
   }
+
+  testVector("getUTF8String should return null for null slot", 4, StringType) {
```

Comment:
we already have test cases for each type, can we just change the existing test cases a little to add this null check?

Reply:
That sounds ok to me. I'll commit the change by tonight.

Reply:
Seems we don't have individual tests for binary type and decimal type?